packages/dd-trace/test/config.spec.js

```diff
@@ -4,6 +4,7 @@ require('./setup/tap')
 
 const { expect } = require('chai')
 const { readFileSync } = require('fs')
+const { clear: clearExtraServices } = require('../src/service-naming/extra-services')
```

Comment:
or you know:
```suggestion
const extraServices = require('../src/service-naming/extra-services')
extraServices.clear()
```
your call